apps/volunteers/views.py

```diff
@@ -30,8 +30,13 @@
     has_authenticated_profile,
     login_url=reverse_lazy('accounts:complete_profile')
 )
-def index(request):
-    return HttpResponse('Hello there!')
+def volunteers(request):
+    volunteer=Volunteer.objects.all()
```

Comment:
Using `volunteers` variable name will be more appropriate here as it contains a list of volunteers.

Reply:
Yup, Done this.